### PR TITLE
Update environment.yml

### LINKED ISSUE #20: GPU not detected
## Information

The problem arises in chapter:

* [* ] Introduction
* [ ] Text Classification
* [ ] Transformer Anatomy
* [ ] Multilingual Named Entity Recognition
* [ ] Text Generation
* [ ] Summarization
* [ ] Question Answering
* [ ] Making Transformers Efficient in Production
* [ ] Dealing with Few to No Labels
* [ ] Training Transformers from Scratch
* [ ] Future Directions

## Describe the bug

After creating the conda env using `environment.yml`, the local GPU cannot be detected as shown below. 
No GPU was detected! This notebook can be *very* slow without a GPU 🐢
Using transformers v4.11.3
Using datasets v1.16.1

$ python
Python 3.9.7 (default, Sep 16 2021, 13:09:58) 
[GCC 7.5.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import torch
>>> torch.cuda.is_available()
False

## To Reproduce

1. conda env create -f environment.yml
2. conda activate book
3. jupyter notebook 01_introduction.ipynb

## Expected behavior
torch.cuda.is_available() should be true.

### PR DESCRIPTION
**Pull request by JingchaoZhang**
fix GPU not found error

Closes #20

### CODE REVIEW

**Comment by lewtun** [comment]
Thank you for the fix 🤗 !
